pilot/helpers/Project.py

```diff
@@ -121,7 +121,7 @@ def start(self):
                     print('yes/no', type='button')
                     should_overwrite_files = styled_text(
                         self,
-                        f'Do you want to overwrite the dev step {self.args["skip_until_dev_step"]} code with system changes? Type y/n',
+                        f"Can I overwrite any changes that you might have made to the project since last running GPT Pilot (y/n)?",
```

Comment:
Unrelated to other changes, this is just a rewording of the message because the previous one is really unclear even to people who work on GPT Pilot (ie. me :smile: ).